marklogic-data-hub-central/ui/src/components/facet/facet-element.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import styles from './facet.module.scss';
+import { numberConverter } from '../../util/number-conversion';
+import { stringConverter } from '../../util/string-conversion';
+import { MLTooltip, MLCheckbox } from '@marklogic/design-system';
+import { OverflowDetector } from 'react-overflow';
+
+
+export const FacetName = (props) => {
```

Comment:
I thought there was a RTL test added for this initially, now I dont find it. Was it removed?

Reply:
I was having issues testing tooltip functionality using RTL. Probably related to the CSS visibility properties. I will do research and try to add the test in my next pr.